components/molecule/textareaField/demo/index.js

```diff
@@ -6,6 +6,10 @@ import withState from './withState'
 
 const MoleculeTextareaFieldWithState = withState(MoleculeTextareaField)
 
+const computeExceedLengthErrorText = exceedAmmount => {
+  return `Has excedido ${exceedAmmount} carácteres el texto máximo.`
```

Comment:
all documentation must be included in english plz

Reply:
Sorry!